lib/generators/tenant_level_security/templates/initializer.rb

```diff
@@ -0,0 +1,6 @@
+# Pass current tenant ID to `TenantLevelSecurity.current_tenant_id` block
+# using ActiveSupport::CurrentAttributes, RequestStore, etc...
+# TenantLevelSecurity.current_tenant_id { RequestStore.store[:current_tenant_id] }
```

Comment:
In this day and age, it may be better to recommend `CurrentAttributes` as the default.

Reply:
add comment 1a06ee65a472be9b3c888d92b6ca114b65eb5d4c